e2e/spec/authoring-api/bulk-keys-upload.test.js

```diff
@@ -7,8 +7,8 @@ const Rx = require('rx');
 const nconf = require('nconf');
 
 let should = chai.should();
-const authoringApiRequest = supertest(nconf.get('AUTHORING_URL'));
-const tweekApiRequest = supertest(nconf.get('TWEEK_API_URL'));
+const authoringApiRequest = supertest(nconf.get('AUTHORING_URL') || 'http://localhost:4005');
```

Comment:
the best practice to add default values is when configuring nconf, that way whenever you request that configuration you will always get the same value.
you can see an example in our e2e tests:
https://github.com/Soluto/tweek/blob/master/e2e/wdio.conf.js